diff --git a/node_modules/appium-xcuitest-driver/build/index.d.ts b/node_modules/appium-xcuitest-driver/build/index.d.ts
new file mode 100644
index 0000000..c26a899
--- /dev/null
+++ b/node_modules/appium-xcuitest-driver/build/index.d.ts
@@ -0,0 +1,5 @@
+export { XCUITestDriver };
+export * as doctor from "./lib/doctor/checks";
+export default XCUITestDriver;
+import { XCUITestDriver } from './lib/driver';
+//# sourceMappingURL=index.d.ts.map
\ No newline at end of file
